### PR TITLE
Task 2 - Servereless deployment - S3 - CloudFront

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
React SPA - Serverless deployment to S3 bucket and CloudFront Distribution
## What was done
1. React SPA forked.
2. Added and configured serverless and serverless-finch plugin.
3. 'npm start' script is starting the local app perfectly and FE is working as expected. 
4. 'npm build' script has built the artifacts perfectly. 
5. Three visible changes have been made in the app -
    a. changed store name
    b. changed account icon
    c. changed the color of the product title.
    d. added on hover effect on each of the cards.
6. Deployed build on S3 bucket(through scripts) and CloudFront Distribution(through scripts)
7. S3 bucket URL is not accessible
8. CloudFront Distribution Link is accessible and the updated app can be viewed through the link.

## Link to forked repository
https://github.com/Terala-Priyanka/shop-react-redux-cloudfront

### CODE REVIEW

File: serverless-single-page-app-plugin/index.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
+'use strict';
```
Comment (cetrix13) [inline serverless-single-page-app-plugin/index.js]: There is a duplication. The same file exists in a directory '.serverless_plugins'. Please keep one copy of this file.

**Review by cetrix13 — CHANGES_REQUESTED** [review]

**Comment by cetrix13** [comment]
**Peer-review:**

- [x] 4 - In addition to the previous work a CloudFront distribution...
- [x] 5 - Serverless-finch and serverless-single-page-app plugins are added and configured. ... 

Good job!

File: serverless-single-page-app-plugin/index.js (hunk 1 above)
Comment (Terala-Priyanka) [reply serverless-single-page-app-plugin/index.js]: @cetrix13 
To my understanding I need to remove .serverless_plugins folder, as package.json has a mention of 
"serverless-single-page-app-plugin": "file:./serverless-single-page-app-plugin",. 
Could you confirm if I am correct here?

Comment (cetrix13) [reply serverless-single-page-app-plugin/index.js]: Yes, you're right.

Comment (Terala-Priyanka) [reply serverless-single-page-app-plugin/index.js]: @cetrix13 
I have removed it. Thank you!

**Pull request merged**